src/Playwright.Tests/ScreencastTests.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Microsoft.Playwright.Tests
 {
+    /*
```

Comment:
Should we leave the test, but skip it, and comment the code, so we don't forget?

Reply:
I have an issue for that. It will be my next task

Reply:
Ok, that's fine then.